install.markdown

```diff
@@ -3,37 +3,29 @@ layout: page
 title: Install IntelliJ Rust
 ---
 
-## Getting alpha builds
+## Alpha builds
```

Comment:
Maybe it's time to get rid of `alpha` in docs?
It was true three years ago but now we've become much more featured and support the most part of the language